README.rst

```diff
@@ -40,6 +40,8 @@ Installing via `pip <https://pip.pypa.io/>`_:
 .. code-block:: console
 
     $ pip install cterasdk
+    $ pip install cterasdk --trusted-host pypi.org --trusted-host files.pythonhosted.org  # [SSL: CERTIFICATE_VERIFY_FAILED]
```

Comment:
This way it seems like the user needs to run all three commands. I think you should split it into sections to make it clear that they are mutually exclusive

Reply:
Done